nginx/tests/integration/charms/k8s/pyproject.toml

```
@@ -0,0 +1,12 @@
+[project]
+name = "test"
+version = "0.1.0"
+description = "Test charm for integration tests"
+requires-python = ">=3.8"
```

`uv lock` of the assembled charm for integration tests will fail because of this if we drop Python 3.8.
```suggestion
requires-python = ">=3.10"
```
Other than that, dropping Python 3.8 seems to work fine in this dummy PR https://github.com/canonical/charmlibs/pull/150